packages/node-app/src/swagger.json

```diff
@@ -0,0 +1,634 @@
+                    },
+                    "5XX": {
+                        "description": "Unexpected error"
+                    }
+                }
+            }
+        },
+        "/giveaways/{giveawayId}/participants/{participantId}": {
+            "put": {
+                "tags": ["giveaways"],
+                "description": "Update participant state",
+                "parameters": [
+                    {
+                      "name": "giveawayId",
+                      "in": "query",
+                      "description": "ID of the giveaway",
+                      "required": true
+                    },
+                    {
+                        "name": "participantId",
+                        "in": "query",
+                        "description": "ID of the participant",
+                        "required": true
+                    },
+                    {
+                        "name": "state",
+                        "in": "body",
+                        "description": "New state of the participant",
+                        "required": true
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "type": "object",
+                                    "example": {
+                                        "message": "Participant updated successfully"
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "401": {
+                        "description": "Invalid Authentication"
+                    },
+                    "4XX": {
+                        "description": "Invalid or missing information"
+                    },
+                    "5XX": {
+                        "description": "Unexpected error"
+                    }
+                }
+            }
+        },
+        "/locations": {
+            "get": {
+                "tags": ["locations"],
+                "description": "Get all locations",
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "type": "array",
+                                    "items": {
+                                        "$ref": "#/components/schemas/LocationItem"
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "401": {
+                        "description": "Invalid Authentication"
+                    },
```

Comment:
Maybe for admin endpoints, we should also specify the 403 forbidden since we also have the 401

Reply:
For what I can see in the code the `verifyAdmin` middleware returns a 401 for cases where the user doesn't have the required role, not 403

Reply:
In that case I think we should update it to 403 in the future, it is probably more appropriate